vaadin-icons-flow-integration-tests/src/test/java/com/vaadin/flow/component/icon/tests/IconIT.java

```diff
@@ -20,11 +20,15 @@
 import org.junit.Assert;
 import org.junit.Test;
 import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptException;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
 import com.vaadin.flow.component.icon.VaadinIcon;
 import com.vaadin.flow.component.icon.demo.IconView;
 import com.vaadin.flow.demo.ComponentDemoTest;
+import org.openqa.selenium.chrome.ChromeDriver;
```

Comment:
check and delete unused imports like `ChromeDriver`

Reply:
done